composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
@Nicofuma Oh sure, I was saying that other deps required by extensions could be problematic, not `phpbb/phpbb`. Need to find a way to proxy them soon unless I've missed something?

Also '~3.2.0@dev' should be the soft requirement, not the hard one. I'll explain in a little bit.